index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
```suggestion
<h2 hidden>Game over, refresh to play again! Play again if you want to win</h2>
```
added a message play agagin if you want to win to provoke the player and make them play again